.golangci.yml

```diff
@@ -0,0 +1,65 @@
+run:
+  deadline: 1m
+
+linters:
+  enable-all: true
+  disable:
+    - gocyclo
+    - golint
+    - maligned
+    - errcheck
+    - staticcheck
+    - dupl
+    - ineffassign
+    - interfacer
+    - unconvert
+    - goconst
+    - unparam
+    - nakedret
+    - lll
+    - gochecknoglobals
+    - govet
```

Comment:
We do not pass govet? Wasn't this enabled in metalinter? 
https://github.com/tendermint/tendermint/pull/3261/commits/da33dd04cc90f60f339afa8182f59294e67d151b#diff-b67911656ef5d18c4ae36cb6741b7965L252

Reply:
probably because of the older version

Reply:
Weird, I thought govet is just part of the golang distribution (the `go vet` command).

Reply:
Or maybe gometalinter was broken